example_template.md

```diff
@@ -41,76 +44,116 @@ git clone https://github.com/jina-ai/examples
+### 🏃 Step 3. Index your data
+In this step, we will index our data.
 
-Most Jina applications will use two Flows. One for Indexing and one for Querying.
+*Here describe the Index Flow. Be as specific as possible in describing how this Index Flow works and what is its input. You are encouraged to use code snippets, images, or whatever helps to clarify.*
 
-### Step 1: Index your data
+```
+python app.py -t index (replace as necessary)
+```
 
-Describe the Index Flow. Be as specific as possible. You are encouraged to user code snippets, images, or whatever helps to clarify.
+If you see the following output, it means your data has been correctly indexed.
 
-### Step 2: Search your data
+```
+Flow@5162[S]:flow is closed and all resources are released, current build level is 0
+```
 
-Describe the Query Flow. Be as specific as possible. You are encouraged to user code snippets, images, or whatever helps to clarify.
+### 🔎 Step 4: Query your data
+Next, we will deploy our query Flow.
+
+*Here describe the Query Flow. Be as specific as possible in describing how this Query Flow works and what is its input. You are encouraged to use code snippets, images, or whatever helps to clarify.*
+
+Run the query Flow in your terminal like this:
+
+```
+python app.py -t query (replace as necessary)
+``` 
+______
+
+## 📉 Understanding your results
+*Here include a short description of the results and how to interpret them if needed.*
 
 ## 🌀 Flow diagram
+This diagram provides a visual representation of the Flows in this example; Showing which executors are used in which order.
 
-Show the Flow for this example.
+*Here Show the Flow for this example.*
 
-## 🌟 Results
+## 📖 Optional: Extra information useful for the user
 
-Short description of the results and how to interpret them if needed.
+*Use this section to add extra information you think the user could benefit from.
+QueryLanguage, Faiss, Annoy for example.*
 
-## 🧞 Optional: Extra information useful for the user
+## 🔮 Overview of the files
 
-You can use this section to add extra information you think the user could benefit from.
-QueryLanguage, Faiss, Annoy for example. 
+*Add a list with all folders/files in the example:*
 
-## 💫 Deploy with Docker
+|                      |                                                                                                                  |
+| -------------------- | ---------------------------------------------------------------------------------------------------------------- |
+| 📂 `flows/`          | Folder to store Flow configuration                                                                               |
+| --- 📃 `index.yml`     | YAML file to configure indexing Flow                                                                             |
```

Comment:
I really like the use of emojis :)

Reply:
Over time I will transition readmes to use more and more emojis until there are no normal characters remaining